### ISSUE TITLE
rustup doc with open error

### ISSUE THREAD

**Issue by h3n4l** [rust-lang/rust#95586]
I tried this code:

```bash
rustup doc
```

I expected to see this happen: 
Open index.html in the web browser or other correct behavior

Instead, this happened:
```bash
error: couldn't open browser: command 'open' did not execute successfully; exit status: 1
command stderr:
The file /Users/h3n4l/.rustup/toolchains/stable-aarch64-apple-darwin/share/doc/rust/html/index.html does not exist.
```

### Meta

`rustc --version --verbose`:
```
rustc 1.59.0 (9d1b2106e 2022-02-23)
binary: rustc
commit-hash: 9d1b2106e23b1abd32fce1f17267604a5102f57a
commit-date: 2022-02-23
host: aarch64-apple-darwin
release: 1.59.0
LLVM version: 13.0.0
```

**Comment by h3n4l** [comment]
It seems like that there was no a folder named `html` in the path `~/.rustup/toolchains//stable-aarch64-apple-darwin/share/doc/rust/`:
```bash
ls -l /Users/h3n4l/.rustup/toolchains/stable-aarch64-apple-darwin/share/doc/rust
total 80
-rw-r--r--  1 h3n4l  staff   9322 Apr  2 18:03 COPYRIGHT
-rw-r--r--  1 h3n4l  staff   9723 Apr  2 18:03 LICENSE-APACHE
-rw-r--r--  1 h3n4l  staff   1023 Apr  2 18:03 LICENSE-MIT
-rw-r--r--  1 h3n4l  staff  10307 Apr  2 18:03 README.md
```

**Comment by ehuss** [comment]
Thanks for the report!  Documentation for that target is not available in 1.59.  This will be fixed in the 1.60 release next week.  Closing as a duplicate of #69525.
